tests/test_validation.py

```diff
@@ -0,0 +1,169 @@
+import shutil
+from pathlib import Path
+
+import great_expectations as ge
+import pandas as pd
+import pytest
+from great_expectations.core.batch import RuntimeBatchRequest
+from great_expectations.exceptions.exceptions import CheckpointNotFoundError
+from prefect import flow
+
+from prefect_great_expectations.validation import (
+    GreatExpectationValidationError,
+    run_checkpoint_validation,
+)
+
+CURRENT_DIR = Path(__file__).parent.resolve()
+CONTEXT_ROOT_DIR = CURRENT_DIR / "great_expectations"
+DATA_PATH = CURRENT_DIR / "data"
+
+
+@pytest.fixture(autouse=True)
+def disable_usage_stats(monkeypatch):
+    monkeypatch.setenv("GE_USAGE_STATS", "FALSE")
+
+
+class TestRunCheckpointValidation:
+    def test_validation_wth_checkpoint_name(self):
+        @flow
+        def test_flow():
+            return run_checkpoint_validation(
+                checkpoint_name="my_checkpoint_pass",
+                data_context_root_dir=CONTEXT_ROOT_DIR,
+            )
+
+        result = test_flow().result().result()
+        assert result.success, "Checkpoint validation should pass"
+
+    def test_validation_with_checkpoint(self):
+        @flow
+        def test_flow():
+            checkpoint = ge.DataContext(
+                context_root_dir=str(CONTEXT_ROOT_DIR)
+            ).get_checkpoint("my_checkpoint_pass")
+            return run_checkpoint_validation(
+                checkpoint=checkpoint, data_context_root_dir=CONTEXT_ROOT_DIR
+            )
+
+        result = test_flow().result().result()
+        assert result.success, "Checkpoint validation should pass"
```

Comment:
Does the string do anything here?
`assert result.success, "Checkpoint validation should pass"`

Reply:
Yes, it displays that message when the assertion fails.